lib/db.js

```diff
@@ -827,6 +827,24 @@ module.exports = function (
     return this.pool.del('/verificationReminders', reminderData)
   }
 
+  DB.prototype.securityEvent = function (event) {
```

Comment:
It might help for clarity in calling code if this methods was called `createSecurityEvent`.